packages/data-transformation/fhir-to-csv/src/setupSnowflake/setupSnowflake.py

```
@@ -0,0 +1,153 @@
+        create_statement += ",\n".join([f"  {col} {get_data_type(col, date_types)}" for col in columns])
+        create_statement += "\n)\n"
+        table_names_and_create_temp_table_statements.append((table_name, create_statement))
+
+    return table_names_and_create_temp_table_statements
+
+def create_temp_tables(cx_id: str):
+    database_name = format_database_name(cx_id)
+    with snowflake.connector.connect(**get_snowflake_credentials(), autocommit=False) as snowflake_conn:
+        snowflake_conn.cursor().execute(f"CREATE DATABASE IF NOT EXISTS {database_name}")
+        snowflake_conn.cursor().execute(f"USE DATABASE {database_name}")
+        snowflake_conn.cursor().execute(f"USE SCHEMA PUBLIC")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)